kubernetes/base/dynamic/test_client.py

```diff
@@ -527,9 +526,8 @@ def test_server_side_apply_api(self):
                             'ports': [{'containerPort': 80,
                                             'protocol': 'TCP'}]}]}}
 
-        body = json.dumps(pod_manifest).encode()
         resp = api.server_side_apply(
-            name=name, namespace='default', body=body,
```

Comment:
> body=body

Does this mean users can no longer pass JSON as the body? Is this considered a breaking change?

Reply:
It would be a breaking change, though I'm not sure that we intended for this to accept a string in the first place. Could add a little bit of logic to server_side_apply to attempt to yaml load the content in order to preserve that behavior if it's what we want

Reply:
> I'm not sure that we intended for this to accept a string in the first place

I agree. I'm okay with calling it out as a breaking change in the release notes. We could provide instruction/util to help users switch/keep the behavior, but the default behavior doesn't have to accept a string.